.travis.yml

```diff
@@ -15,12 +17,15 @@ matrix:
     - os: osx
       osx_image: xcode6.4
       go: 1.8.x
+      env: SKIP_GRAPHVIZ=true
```

Comment:
Does it have to be "true"? I don't think bash have special support for boolean types?

Reply:
It doesn't have to be true -- is there something else you'd recommend?

Reply:
I'd just do "1".

Reply:
Done.